### ISSUE TITLE
Upgrade to Python3

### ISSUE THREAD

**Issue by jameshiebert** [pacificclimate/routomator#1]
Python 2 is deprecated. This package needs to be upgraded to Python 3.

**Comment by rod-glover** [comment]
I somehow came to the conclusion that this package was no longer in use ... likely incorrectly, I now realize, but it's worth asking who uses it.

**Comment by AreliaTW** [comment]
yes I would like to be able to use it. if we can get it working rather soon especially the python generate_fraction_file.py script it would be helpful for the BCSRIF project.

**Comment by AreliaTW** [comment]
it seems James made some pretty good gains yesterday. I am not sure how much more work is required to reach the end goal.

**Comment by jameshiebert** [comment]
I have a [branch named `python3`](https://github.com/pacificclimate/routomator/tree/python3) in progress.
